charts/images.yaml

```diff
@@ -59,8 +59,8 @@ images:
   tag: v0.5.0
 - name: csi-plugin-alicloud
   sourceRepository: https://github.com/kubernetes-sigs/alibaba-cloud-csi-driver
-  repository: registry.eu-central-1.aliyuncs.com/gardener-de/csi-plugin-alicloud
-  tag: v1.18.8-45 # the upstream image is using non-semver tags which is causing issues in the CI/CD pipelines of Gardener, thus the image is replicated and tagged with semver version in another registry (registry.cn-hangzhou.aliyuncs.com/acs/csi-plugin:v1.18.8.45-1c5d2cd1-aliyun).
+  repository: registry.eu-central-1.aliyuncs.com/acs/csi-plugin
+  tag: v1.20.5-ff6490f-aliyun
```

Comment:
It seems sematic version compliant now :) Need to be keep an eye on our CICD pipeline.